opencti-platform/opencti-front/lang/back/fr.json

```diff
@@ -599,6 +599,7 @@
   "Target type": "Type cible",
   "Task filters": "Filtres de tâches",
   "Tasks": "Tâches",
+  "Telemetry manager": "Gestionnaire de télémétrie",
```

Comment:
All other managers in french are translated as "Manager". 
"Manager de télémétrie"

Reply:
fixed